Gemfile

```diff
@@ -8,6 +8,7 @@ gem 'bump'
 gem 'rack'
 gem 'rake'
 gem 'rspec', '~> 3.11'
+gem 'rubocop-capybara', github: 'rubocop-hq/rubocop-capybara'
```

Comment:
It looks like the rubocop-capybara organization is rubocop-hq instead of rubocop.
Sorry if this is the correct procedure for adding a new extension gem.

Reply:
I don't have permissions to create new repos in `rubocop` 🤷

Reply:
Oh that's right. My apologies 🙇‍♂